rskj-core/src/main/java/org/ethereum/facade/EthereumImpl.java

```diff
@@ -36,13 +37,15 @@ public class EthereumImpl implements Ethereum {
     private final CompositeEthereumListener compositeEthereumListener;
     private final Blockchain blockchain;
     private final GasPriceTracker gasPriceTracker;
+    private final Double minGasPriceMultiplier;
 
     public EthereumImpl(
             ChannelManager channelManager,
             TransactionGateway transactionGateway,
             CompositeEthereumListener compositeEthereumListener,
             Blockchain blockchain,
-            GasPriceTracker gasPriceTracker) {
+            GasPriceTracker gasPriceTracker,
+            Double minGasPriceMultiplier) {
```

Comment:
does it really need to be a `Double`? this could lead to `NullPointerException`'s later when used..

Reply:
I didn't want to make it mandatory, so if you want to use the implementation for testing purposes it is not needed to set a random value there. Also the other members are objects that can lead to a NPE as well and the object it is only created once. In any case if you think is better to use the primitive value I'll update it.

Reply:
well, `null`'s were a billion-dollar mistake, so that not sure it's a good idea to make that number even bigger ;) . 

Nonetheless, while using the scalar type instead of the class, in tests you can always provide some default / dummy value, eg. `1.0`. In this case your object won't be in a `inconsistent` state, which can crash on some occasions.

Also if the `null`'s are really needed and do not lead to objects being in the "inconsistent" state, then you can always mark them, for instance, with the `@Nullable` annotation, so that static analysis tools can later help with detecting unexpected code execution flows.

Reply:
I just wanted to be practical but you are right, so code updated!